string task
---
bool success
string message 